ocs_ci/helpers/ocs_metrics_exporter_helpers.py

```
@@ -0,0 +1,1045 @@
+    """
+    Add an IP address to the Ceph OSD blocklist.
+
+    Args:
+        toolbox_pod (Pod): Ceph toolbox pod
+        ip_address (str): IP address to blocklist
+
+    Returns:
+        str: Command output
+    """
+    cmd = f"ceph osd blocklist add {ip_address}:0/0"
+    logger.info(f"Adding {ip_address} to Ceph blocklist")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)